src/main/java/de/slimecloud/slimeball/features/staff/AbsenceCommand.java

```diff
@@ -0,0 +1,44 @@
+package de.slimecloud.slimeball.features.staff;
+
+import de.mineking.discordutils.commands.ApplicationCommand;
+import de.mineking.discordutils.commands.ApplicationCommandMethod;
+import de.mineking.discordutils.commands.option.Option;
+import de.slimecloud.slimeball.config.GuildConfig;
+import de.slimecloud.slimeball.main.SlimeBot;
+import lombok.RequiredArgsConstructor;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.Instant;
+
+@RequiredArgsConstructor
+@ApplicationCommand(name = "absence", description = "Abwesenheit an/abmelden")
+public class AbsenceCommand {
```

Comment:
You should add a command permission entry for this to restrict this to @Rat

Reply:
i would do this discord integration permissions

Reply:
I would do both